dhall-docs/src/Dhall/Docs.hs

```diff
@@ -213,10 +256,16 @@ defaultMain :: Options -> IO ()
 defaultMain Options{..} = do
     resolvedPackageDir <- Path.IO.resolveDir' packageDir
     resolvedOutDir <- Path.IO.resolveDir' outDir
-    dhallFiles <- getAllDhallFiles resolvedPackageDir
+
+    -- Directories on the `path` modules always ends in "/", so we have
+    -- to remove last one
```

Comment:
I don't understand which step this comment refers to.

Reply:
I forgot to move that comment. Updated